rbac/server/blockchain_transactions/role_transaction.py

```diff
@@ -138,18 +323,18 @@
             list: extended list of transactions for batch submission
     """
     conn = await create_connection()
-    roles = (
+    role_admins = (
```

Comment:
it's a list of role_admin objects, not role objects, so isn't this more accurate?

Reply:
Yeah they are role_admin table entries, but the variable naming could be confusing because I first thought why were you getting a list of role admins (like for a role) instead of grabbing roles that the user is a role admin of. 

Perhaps just name it role_admins_entry then?

Reply:
In the context of the function this will naming convention will work.